### ISSUE TITLE
docker image build broken: /bin/sh: 1: npm: not found

### ISSUE THREAD

**Issue by bagage** [osm-fr/osmose-frontend#152]
When using `cd docker && docker-compose up -d`, it fails at step 19/22:

```
Step 17/22 : ADD ./webpack.config.js /data/project/osmose/frontend
 ---> Using cache
 ---> 02c965ed0481
Step 18/22 : ADD ./static /data/project/osmose/frontend/static
 ---> 422180f40b44
Step 19/22 : RUN npm install &&     npm run build
 ---> Running in d68342840ed7
/bin/sh: 1: npm: not found
ERROR: Service 'frontend' failed to build: The command '/bin/sh -c npm install &&     npm run build' returned a non-zero code: 127
```

**Comment by jocelynj** [comment]
I think the issue is because docker base image python:2.7 has switched to buster, which includes a newer version of nodejs, which is more recent than version 9 from deb.nodesource.com.

Can you try changing the line https://github.com/osm-fr/osmose-frontend/blob/master/docker/Dockerfile#L11 to `setup_12.x` or `setup_10.x`?

**Comment by bagage** [comment]
Thanks @jocelynj, it indeed does. I've used node 10. The fix is in #151!
